app/controllers/comparisons/change_in_energy_use_since_joined_energy_sparks_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Comparisons
+  class ChangeInEnergyUseSinceJoinedEnergySparksController < Shared::ArbitraryPeriodController
+    def key
+      :change_in_energy_use_since_joined_energy_sparks
+    end
+
+    def set_headers
+      super
+      @include_previous_period_unadjusted = false
```

Comment:
don't need the rest if call super after?

Reply:
The base class will reset the variable to true, so the headers will end up wrong. I've removed the call to super, which was there to ensure that the `BaseController` version was called and just overridden the method entirely